### ISSUE TITLE
fixing uuid not found problem (works only in chrome)

### ISSUE THREAD

**Issue by sweeeetch** [Lyfhael/DeleteTweets#24]
it should appear like this :
![Screenshot from 2023-12-16 21-41-17](https://github.com/Lyfhael/DeleteTweets/assets/103188197/8c9206e5-7ca8-449c-b5fb-ab824d394420)
to get the uuid you head to application in menu : 
![Screenshot from 2023-12-16 21-42-03](https://github.com/Lyfhael/DeleteTweets/assets/103188197/e59008d7-cc11-469a-8de2-924f8d6bce00)
then go to cookies (storage section) and click twitter ....:

![Screenshot from 2023-12-16 21-42-34](https://github.com/Lyfhael/DeleteTweets/assets/103188197/30791993-203d-4928-8c90-2bfc486b3a97)

next go to pesonal...... and copy its value :
![Screenshot from 2023-12-16 21-42-50](https://github.com/Lyfhael/DeleteTweets/assets/103188197/92f19297-88e1-4ff3-86e7-5f823a252fc7)

and there you go, this is your uuid.
(it worked for me)

source : https://help.uberflip.com/hc/en-us/articles/360041466972-Find-Your-Uberflip-Analytics-Identifier-UUID-

**Comment by eleius** [comment]
I don't have a personalization_id either... [Others](https://github.com/Lyfhael/DeleteTweets/issues/10) suggested you can use a random value, but it doesn't work for me.

**Comment by Marouan-El-Haddad** [comment]
> I don't have a personalization_id either... [Others](https://github.com/Lyfhael/DeleteTweets/issues/10) suggested you can use a random value, but it doesn't work for me.

I had the same problem, but it ended up working if I used the guest_id :) Hopefully it can work for you! (guest_id is same location as personalization_id)

**Comment by hinetapora** [comment]
yes i did not need uuid - it just worked.  please update the readme.  and thanks!!!
